CHANGES.md

```diff
@@ -9,6 +9,8 @@ To be released.
 ### Backward-incompatible API changes
 
 ### Backward-incompatible network protocol changes
+ -  If the count of blocks `Swarm<T>` have received is less than
+    the `Swarm<T>.FindNextHashesChunkSize`, `Swarm<T>` will no longer retry.  [[#1112]]
```

Comment:
As `Swarm<T>.FindNextHashesChunkSize` is `internal`, this should be like instead:

```suggestion
 -  `Swarm<T>` became no longer retry when `Swarm<T>` receives less than 500
    blocks.  [[#1112]]
```